src/main/java/org/tomitribe/auth/signatures/Verifier.java

```diff
@@ -83,14 +83,20 @@ public Verifier(final Key key, final Signature signature, final Provider provide
     }
 
     public boolean verify(final String method, final String uri, final Map<String, String> headers) throws IOException, NoSuchAlgorithmException, SignatureException {
-
+        try {
+            signature.verifySignatureValidityDates();
+        } catch (Exception e) {
```

Comment:
can be final

Reply:
hmm, there is no variable here. Did I miss something?

Reply:
This is no longer applicable because we let verifySignatureValidityDates throw an exception.